src/wdio/services/TerraCommands/visual-regression.js

```diff
@@ -96,24 +96,13 @@ const getTestDescription = matchType => (
   matchType === 'withinTolerance' ? 'be within the mismatch tolerance' : 'match screenshot exactly'
 );
 
-/**
-* A mocha-chai convenience test case to determines the screenshot options and test names needed to
-* capture screenshots of a specified element and assert the screenshot comparision results are
-* either within the mismatch tolerance or are an exact match.
-* @property {Array} testArguments - The test arguments passed to the `matchScreenshotWithinTolerance`
-*    or `matchScreenshotExactly` methods.
-* @property {String} matchType - Specifies the type of matchReference assertion. Either 'withinTolerance'
-*    or 'exactly'.
-*/
-const matchScreenshot = (testArguments, matchType) => {
-  const { name, selector, options } = determineScreenshotOptions(...testArguments);
-
+const matchScreenshotImplementation = (name, matchType, selector, options) => {
```

Comment:
Some of this is getting a little layered. I might be easier to understand if we can someone associate this name with the `it` block or the fact its the VR / screenshot comparison test block

Reply:
Also we should document this method

Reply:
Good call.  I wasn't sure what to call this.  I like referencing that this is the actual it block.